File: Infosys.PackXPrez.Services/Models/Registration.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Infosys.PackXPrez.Common
{
    public class Registration
    {
        [Required]
        public string emailId;
        [Required]
        public string shippingType;
        [Required]
        public short length;
        [Required]
        public short breadth;
        [Required]
        public short height;
        [Required]
        public int weight;
        [Required]
        public string deliveryType;
        [Required]
        public string timeSlot;
        [Required]
        public int pickAddressId;
        [Required]
        public string deliveryAddress;
        [Required]
        public bool packingService;
    }
}

File: PaxkXPrezApp/src/app/packXPrez-interfaces/registration.ts
export interface IRegistration {
  emailId: string,
  shippingType: string,
  length: number,
  breadth: number,
  height: number,
  weight: number,
  deliveryType: string,
  timeSlot: string,
  pickAddressId: number,
  deliveryAddress: string,
  packingService: boolean
}
